nemo/collections/asr/metrics/bleu.py

```diff
@@ -116,7 +116,7 @@ def __init__(
                 encoder_hidden_states=predictions,
                 encoder_input_mask=predictions_mask,
                 decoder_input_ids=input_ids,
-                return_hypotheses=False,
+                return_all_hypotheses=False,
```

Comment:
Why do we want to make this pedantic change in the name and break older inference ? Revert please

Reply:
We tried to handle all cases but will revert this for now as it includes other breaking changes